### PR TITLE
scratchblocks scroll

### PR DESCRIPTION
**Pull request by RedGuy12**
**before**
image

**after**
image

#1222

### CODE REVIEW

**Comment by GrahamSH-LLK** [comment]
Maybe a styled scrollbar? That one looks a little out of place

**Comment by RedGuy12** [comment]
> Maybe a styled scrollbar? That one looks a little out of place

it is styled by dark mode
image
just the first screenshot was taken from the preview iframe which dark mode doesn't have access to

**Comment by TheColaber** [comment]
This was already merged tho...

**Comment by RedGuy12** [comment]
> This was already merged tho...

it was reverted #1237

and I was too lazy to make a new one

**Comment by mxmou** [comment]
> just the first screenshot was taken from the preview iframe which dark mode doesn't have access to

It does have access to it (using a userscript). If its scrollbar CSS isn't injected into the preview then that's a bug.

**Comment by RedGuy12** [comment]
> > just the first screenshot was taken from the preview iframe which dark mode doesn't have access to
> 
> It does have access to it (using a userscript). If its scrollbar CSS isn't injected into the preview then that's a bug.

looks like it's not working

**Comment by mxmou** [comment]
> looks like it's not working

You're right. I'll make an issue about this.

File: addons/full-signature/addon.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Show full areas",
-  "description": "Shows the full signature of users on the forums, and shows full What I've been doing on profiles.",
+  "description": "Shows the full signature of users on the forums, stops Scratchblocks from being cut off at the edge of the screen, and shows full What I've been doing on profiles.",
```
Comment (WorldLanguages) [inline addons/full-signature/addon.json:3]: Not sure on what we should do with this description. We never updated it to reflect we support scroll for "What's happening?". Listing every feature in the description is not very useful

Comment (mxmou) [reply addons/full-signature/addon.json:3]: Maybe something like *Makes some elements with limited height resizable or scrollable, such as user signatures on the forums and What I've been doing on profiles.*

Not really related, but should it be "on the forums" or "in the forums"? Different addons don't seem to agree about this.

Comment (RedGuy12) [reply addons/full-signature/addon.json]: done

**Pull request merged**